### ISSUE TITLE
Wrongly reporting missing semicolon for parent reference

### ISSUE THREAD

**Issue by chodorowicz** [rossPatton/stylint#288]
I'm using `box-sizing: border-box;` rule and stylint reports missing semicolon on parent references.
Both CLI and Atom plugin.
```
Stylint version: 1.3.8

Warning: missing semicolon
File: app/stylus/main.styl
Line: 31: &.is-popupVisible
```

![image](https://cloud.githubusercontent.com/assets/119451/14913528/f2f1140e-0e03-11e6-85be-bc4bfc7130a7.png)

**Comment by rossPatton** [comment]
I can't replicate this issue, try upgrading to 1.5.0?

**Comment by chodorowicz** [comment]
I'm getting missing semicolon now on HTML tags

![image](https://cloud.githubusercontent.com/assets/119451/16224828/17932350-37a4-11e6-9fe7-1faa37dead14.png)

files
https://gist.github.com/chodorowicz/981f8fd41fd96057c7728c352c86c5cf
